app/templating/metadata_template_preprocessor.py

```diff
@@ -18,17 +19,19 @@ def build_metadata(self, schema):
         return render_data
 
     def _build_respondent_meta(self):
-        if self._get_metadata():
-            respondent_id = self._get_metadata().ru_ref
-            name = self._get_metadata().ru_name
-            trading_as = self._get_metadata().trad_as
+        metadata = self._get_metadata()
+
+        if metadata:
+            respondent_id = metadata["ru_ref"]
+            name = metadata["ru_name"]
+            trading_as = metadata["trad_as"]
         else:
             respondent_id = None
             name = None
             trading_as = None
 
         respondent_meta = {
-            "tx_id": convert_tx_id(self._get_metadata().tx_id),
+            "tx_id": convert_tx_id(metadata["tx_id"]),
```

Comment:
should this move to inside the 'if metadata' above like the other values (i.e. will probably fall over if metadata is None)?